pkg/asset/installconfig/vsphere/validation.go

```diff
@@ -2,46 +2,79 @@ package vsphere
 
 import (
 	"context"
+	"fmt"
+	"strings"
 	"time"
 
 	"github.com/pkg/errors"
-	"github.com/vmware/govmomi/find"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
 	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/vsphere"
 	vspheretypes "github.com/openshift/installer/pkg/types/vsphere"
 	"github.com/openshift/installer/pkg/types/vsphere/validation"
 )
 
 // Validate executes platform-specific validation.
 func Validate(ic *types.InstallConfig) error {
-	allErrs := field.ErrorList{}
 	if ic.Platform.VSphere == nil {
 		return errors.New(field.Required(field.NewPath("platform", "vsphere"), "vSphere validation requires a vSphere platform configuration").Error())
 	}
 
-	allErrs = append(allErrs, validation.ValidatePlatform(ic.Platform.VSphere, field.NewPath("platform").Child("vsphere"))...)
+	p := ic.Platform.VSphere
+	vim25Client, _, err := vspheretypes.CreateVSphereClients(context.TODO(),
+		p.VCenter,
+		p.Username,
+		p.Password)
+
+	if err != nil {
+		return errors.New(field.InternalError(field.NewPath("platform", "vsphere"), errors.Wrapf(err, "unable to connect to vCenter %s.", p.VCenter)).Error())
+	}
+
+	finder := NewFinder(vim25Client)
+	return validateVsphere(finder, ic)
+}
 
+func validateVsphere(finder VSphereClient, ic *types.InstallConfig) error {
```

Comment:
The "Vsphere" part is redundant with the package name.
```suggestion
func validate(finder VSphereClient, ic *types.InstallConfig) error {
```

Reply:
just validate is not allowed.
```
validate already declared through import of package validate ("github.com/openshift/installer/pkg/validate")
```